### ISSUE TITLE
make install doesn't update icon cache

### ISSUE THREAD

**Issue by marcelpaulo** [zealdocs/zeal#922]
I've just built zeal from source (30c85bb13679036e8e81b050d01be13d383cf0fe) on Xubuntu 17.10, using the default prefix (_/usr/local_), and the icon cache is not updated. After running `make install`, I updated the cache running:

`sudo gtk-update-icon-cache -f -t /usr/local/share/icons/hicolor`

I imagine the cmake config files should be ammended to include that.

**Comment by trollixx** [comment]
We should use [ECMInstallIcons](https://api.kde.org/ecm/module/ECMInstallIcons.html).

**Comment by trollixx** [comment]
ECM supports icon cache updates since v5.15.0 ([commit](https://github.com/KDE/extra-cmake-modules/commit/12908eafb494ed32a96a5573f0446cc1528265ca), (review)[https://git.reviewboard.kde.org/r/125192/]).

**Comment by marcelpaulo** [comment]
Indeed, the icon cache is being updated now. Removed _/usr/local/share/icons/hicolor/icon-theme.cache_, rebuilt/reinstalled zeal, and the icon cache was regenerated. Thanks !

**Comment by trollixx** [comment]
@marcelpaulo Thanks for verifying the fix! I'll release v0.6.1 shortly.
